python/cuml/cuml/tests/experimental/accel/estimators_hyperparams/test_accel_kmeans.py

```diff
@@ -100,6 +100,3 @@ def test_kmeans_random_state(clustering_data):
     kmeans2 = KMeans(n_clusters=3, random_state=42).fit(X)
     # With the same random_state, results should be the same
     assert np.allclose(kmeans1.cluster_centers_, kmeans2.cluster_centers_)
-    kmeans3 = KMeans(n_clusters=3, random_state=24).fit(X)
-    # With different random_state, results might differ
-    assert not np.allclose(kmeans1.cluster_centers_, kmeans3.cluster_centers_)
```

Comment:
I think it makes sense to remove this part of the test. It is a bit unclear what it is trying to check because the cluster centers might or might not be the same if we use a different seed. This means that all we can assert here is `maybe_allclose`, which doesn't seem like a useful assertion.

So 👍  to removing it